sycl/source/detail/builtins_geometric.cpp

```diff
@@ -16,26 +16,36 @@
 
 namespace s = sycl;
 namespace d = s::detail;
+namespace cl = s::opencl;
```

Comment:
Note for reviewers: I'm specifically not switching the file to use types which are not tied to OpenCL, because current type aliases and built-ins definition in SYCL 2020 spec is a bit confusing. It is not entirely clear which exact aliases should be there and how should they be defined as it is not entirely clear which built-ins should support which data types.

See KhronosGroup/SYCL-Docs#353 and KhronosGroup/SYCL-Docs#335.

Therefore, I simply updating this file to stop using aliases which don't exist in SYCL 2020 spec anymore, fully preserving the actual types which were used in the file, thus making this a non-functional change.

Reply:
Maybe worth putting it into the code as a comment.